src/azure-cli/setup.py

```diff
@@ -148,6 +139,10 @@
     'jsondiff==1.2.0'
 ]
 
+TESTS_REQUIRE = [
```

Comment:
I think, we don't need this. 

Because `mock` is [merged into Python standard library since Python 3.3](mock is now part of the Python standard library, available as unittest.mock in Python 3.3 onwards), and CLI support Python 3.6+ only.

So, I think we should also change code from `import mock` to `from unittest import mock`

Reply:
Just tried replacing `import mock` with `from unittest import mock`, some tests failed. I believe some code is using the latest feature of mock 4.0, so mock bundled in python 3.6 might not work. @jiasli to comment.